frontend-maven-plugin/.nvmrc

```diff
@@ -0,0 +1,4 @@
+        # comment
```

Comment:
Could we move this file under `src.it` so that it's clear that it's part of the test setup?

Reply:
It's testing more than one level of lookup, in the future where we have subfolder support, yes. We could move all of these under something like "node-version-files"